components/hab/src/command/pkg/uninstall.rs

```diff
@@ -33,21 +39,24 @@ use std::{fs,
 ///     5b. If there are not, we delete it from disk and the graph
 ///
 /// `excludes` is a list of user-supplied `PackageIdent`s.
-/// `services` is a list of fully-qualified `PackageIdent`s which are currently
-///    running in a supervisor out of the `fs_root_path`.
-pub fn start(ui: &mut UI,
-             ident: &PackageIdent,
-             fs_root_path: &Path,
-             execution_strategy: ExecutionStrategy,
-             scope: Scope,
-             excludes: &[PackageIdent],
-             services: &[PackageIdent])
-             -> Result<()> {
+/// `even_if_running` is a flag indictating that the package should be uninstalled even if it is
+/// running.
```

Comment:
This option feels hacky. See here for the [explanantion](https://github.com/habitat-sh/habitat/blob/7240f505a9f5b31f2938ea193df92b507724c73f/components/sup/src/util/pkg.rs#L130) of why it is necessary.